src/juju_doctor/builtin/application-exists.py

```diff
@@ -0,0 +1,89 @@
+"""Application-exists verbatim builtin plugin."""
+
+import logging
+from typing import Dict, Optional
+
+from pydantic import BaseModel, ConfigDict, Field
+from rich.logging import RichHandler
+
+from juju_doctor.artifacts import read_file
+
+logging.basicConfig(level=logging.WARN, handlers=[RichHandler()])
+log = logging.getLogger(__name__)
+
+
+class ApplicationAssertion(BaseModel):
+    """Schema for a builtin Application definition in a RuleSet."""
+
+    model_config = ConfigDict(extra="forbid")
+
+    name: str = Field(alias="application-name")
+    minimum: Optional[int] = Field(None, ge=0)
+    maximum: Optional[int] = Field(None, ge=0)
+
+
+def status(juju_statuses: Dict[str, Dict], **kwargs):
+    """Status assertion for applications existing verbatim.
+
+    >>> status({"0": example_status()}, with_args=example_with_fake_name())  # doctest: +ELLIPSIS
```

Comment:
Not sure what's the idea here.
1. How about `with_` instead of `with_args`?
2. We do we collect everything into a single `with_args` if we already have kwargs here? I.e. should the example read `status({"0": example_status()}, **{"application-name": "alertmanager_fake"})`? I.e. in the yaml the `with:` is vectorized, but juju doctor can still call `status()` individually.